python/mlc_chat/op/gemm.py

```diff
@@ -7,11 +7,12 @@
 from tvm.relax.frontend.nn import op
 
 
-def faster_transformer_dequantize_gemm(
+def faster_transformer_dequantize_gemm(  # pylint: disable=too-many-arguments
```

Comment:
It's always challenging to find a perfect name, but how about renaming this to `ft_gemm.py`?